docs/source/optional_plugins/golang.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Golang Plugin
+=============
+
+This optional plugin enables Avocado to work with tests originally
```

Comment:
Suggestion:

"This optional plugin enables Avocado to list and run tests written using ..."

The idea being that "originally", to me, sounds like tests have to change to work with Avocado.

Reply:
Ok, good.